### ISSUE TITLE
Transactions over 1000 without decimal places leads to misleading comma separator

### ISSUE THREAD

**Issue by mh182** [siddhantgoel/beancount-dkb#206]
I had a booking over 1000€ with no decimal places. This leads to a separator for a thousand (dot `.`) in the beancount file, which is misinterpreted as a comma separator.

```
# Show minimal example - transaction over 1000€ with no decimal places
➜ cat incoming/07-12-2024_Umsatzliste_Visa\ Kreditkarte_wxyz.csv
"Karte";"Visa Kreditkarte";"4930 •••• •••• wxyz"
""
"Saldo vom 07.12.2024:";"-1407,28 EUR"
""
"Belegdatum";"Wertstellung";"Status";"Beschreibung";"Umsatztyp";"Betrag (€)";"Fremdwährungsbetrag"
"23.11.24";"25.11.24";"Gebucht";"Foo Geschäft";"Im Geschäft";"-1.080";""
"23.11.24";"25.11.24";"Gebucht";"Foo Geschäft";"Im Geschäft";"-1.070,10";""

# Extract beancount from DKB credit card export
➜ beancount-dkb-credit extract incoming/*Kreditkarte_wxyz.csv > kreditkarte.beancount
* /home/max/Documents/accounting/haushalt/incoming/07-12-2024_Umsatzliste_Visa Kreditkarte_wxyz.csv ... OK

# Show created beancount file which has a transaction of -1.080 instead of -1080
➜ cat kreditkarte.beancount
;; -*- mode: beancount -*-

**** /home/max/Documents/accounting/haushalt/incoming/07-12-2024_Umsatzliste_Visa Kreditkarte_0284.csv

2024-11-25 * "Foo Geschäft"
  Liabilities:CC:DKB:Max  -1.080 EUR

2024-11-25 * "Foo Geschäft"
  Liabilities:CC:DKB:Max  -1070.10 EUR

2024-12-07 balance Liabilities:CC:DKB:Max                          -1407.28 EUR
```

The same behavior is also reproducible with the `beancount-dkb-ec`.

**Comment by siddhantgoel** [comment]
Which version are you using? I thought that this was fixed in #200 (there's also a test case included in the PR).

**Comment by mh182** [comment]
I use `1.3.0`.

```
➜ uv tool list
beancount-dkb v1.3.0
- beancount-dkb-credit
- beancount-dkb-ec
```

I just have seen that you released `1.4.0` I will try this one and report back if it is reproducible.

**Comment by mh182** [comment]
Just checked with 1.4.0 and the bug doesn't show up anymore. So you can close it.
Thanks for the help.
